installation/resources/readme.md

```diff
@@ -0,0 +1,21 @@
+# Compass Resources
+
+## Installation CRs
```

Comment:
The table lists files containing different CRs and not really CR names themselves. 
Maybe it would be useful to add a sentence describing the table above it? Sth like in my suggestion (feel free to modify it to sth more appropriate). 

```suggestion
## Installation CRs

The following table lists all the Installation custom resource files for Compass:
```